### ISSUE TITLE
Client Panic On Server Response

### ISSUE THREAD

**Issue by pascaldekloe** [taosdata/driver-go#294]
https://github.com/taosdata/driver-go/blob/4ba75396878d2a5bc56f3c7f4c28a93394a108ab/ws/tmq/consumer.go#L536

Can't just panic when we don't like the response of a server. 😄

**Comment by pascaldekloe** [comment]
Response code is lost like this.

```
panic: tmq not init

goroutine 52 [running]:
github.com/taosdata/driver-go/v3/ws/tmq.(*Consumer).Poll(0x140002822c0, 0x7cd)
	/Users/pascal/go/pkg/mod/github.com/taosdata/driver-go/v3@v3.5.6/ws/tmq/consumer.go:536 +0xe38
github.com/mestral/derive.(*Listen).InstructStorageAt(0x1400007e000, {0xf31058f2?, 0x102a84780?, 0x102a84780?})
```

**Comment by huskar-t** [comment]
fixed version v3.5.8
